spec.md

```diff
@@ -365,11 +365,26 @@ PUT `/containers/v1/volumes/{id}/actions/publish`
         ],
         "lun_id": 0,
         "serial_number": "4349bd228896f1236c9ce9006592f26f",
-        "target_name": "iqn.2007-11.com.nimblestorage:group-array1-g3b5de80e54af7a6b"
+        "target_names": "iqn.2007-11.com.nimblestorage:group-array1-g3b5de80e54af7a6b"
     }
 }
 ```
  * Note that `chap_user` and `chap_password` must also be part of the response if CHAP details were provided as part of the Node definition.
+ * `target_names` json field should be separated by comma, if there are multiple target IQN's needs to be passed to CSI Driver.
```

Comment:
@wdurairaj target_names should be returned as an array from CSP and converted to CSV format by CSI driver later.